packages/trusted-contribution/src/trusted-contribution.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Application } from 'probot';
+import {
+  ChecksCreateParams,
+  PullsListFilesResponse,
+  PullsListFilesResponseItem,
+  Response,
+} from '@octokit/rest';
+
+type Conclusion =
+  | 'success'
+  | 'failure'
+  | 'neutral'
+  | 'cancelled'
+  | 'timed_out'
+  | 'action_required'
+  | undefined;
+
+// TODO: add this to a configuration file to be extended per repository.
+const TRUSTED_CONTRIBUTORS = ['renovatebot', 'release-please'];
+function isTrustedContribution(author: string): boolean {
+  return TRUSTED_CONTRIBUTORS.includes(author);
+}
+
+export = (app: Application) => {
+  app.on('pull_request', async context => {
```

Comment:
Is there a way to re-trigger this when the branch is updated?

Reply:
The context payload has an action with opened, reopened, synchronized, etc. In this case it's running on any of those types.

Reply:
Rad

Reply:
Actually, we probably want to limit this - if kokoro removes the label, will it trigger an event and cause this bot to re-add the label?